new-log-viewer/src/utils/data.ts

```diff
@@ -38,7 +38,45 @@ const getMapValueWithNearestLessThanOrEqualKey = <T>(
         map.get(lowerBoundKey) as T;
 };
 
+/**
+ * Creates an array of numbers in a specified range [startNum, startNum + steps].
```

Comment:
This utility function was created with a reference to Python's build-in https://docs.python.org/3.12/library/stdtypes.html#range method.